### DISCUSSION TITLE
How to configure different soundfonts?

### DISCUSSION THREAD

**Discussion by pedrolcl** [pedrolcl/dmidiplayer#18] [Q&A] [answered]
Question posted by @JeffRocchio in [SF ticket #20](https://sourceforge.net/p/dmidiplayer/tickets/20/):

Just installed the app. I am on fedora-40. I have specified the use of fluidsynth as the MIDI output/synth. But I'm not seeing any way to tell the app to tell fluidsynth to use the set of custom soundfont files that I need it to use for playback. Per Linux standards, the soundfont files are in /usr/share/soundfonts.

Is it possible to do this? If so, how would I go about it?

**Comment by pedrolcl** [comment, marked as answer]
See the attached picture. Press the "MIDI Setup" button (1), (or open the menu Settings->MIDI Setup). In the Connections dialog, choose Fluidsynth on both drop down lists. Press the wrench button (2) to open the Fluidsynth driver settings (3). On this dialog, fill the "Sound Font" text box (or press the "..." button to open the file selection dialog (4).

![ksnip_20240622-092534](https://github.com/pedrolcl/dmidiplayer/assets/15184708/d5a83c4a-a05e-4eca-ad7c-d92ac74c41db)

**Comment by JeffRocchio** [comment]
Pedro - got it. Thanks for the timely response. And thanks for this app.

If you are looking for enhancement suggestions - the ability to load several soundfonts would be great as I have a few different ones for different instruments; and sometimes I have MIDI files where I want to use different .sf2 files for different parts (MIDI tracks).

But again, thanks for helping with my question.

**Reply by pedrolcl** [reply]
You are welcome. Please feel free to mark it as answer.

**Reply by JeffRocchio** [reply]
I would do, but you have to have, I think, maintainer or moderator rights to mark a reply as an 'Answer.' So I think in your repo only you can mark it as such Pedro. But I 'approve' your response as the answer if that matters.

**Reply by pedrolcl** [reply]
You can't mark an answer because you are not the questioner. Well, you are indeed in SF but GH doesn't know about that.

**Comment by JeffRocchio** [comment]
Oh, that's right, I did originate the question on Source Forge, lol

Jeffrey Rocchio
***@***.***

"DON’T BELIEVE EVERY THING YOU THINK"
